dcm4che-core/src/main/java/org/dcm4che3/data/Attributes.java

```diff
@@ -2199,6 +2211,12 @@ public boolean addSelected(Attributes other, String privateCreator, int tag) {
     }
 
     public boolean addWithoutBulkData(Attributes other, BulkDataDescriptor descriptor) {
+        List<ItemPointer> itemPointers = new ArrayList<>();
+        itemPointers.addAll(Arrays.asList(other.itemPointers()));
```

Comment:
This particular item is an O(n) call, but the addWithoutBulkData should be called externally very occasionally, so that shouldn't be too expensive overall.